src/reducers/filter.js

```diff
@@ -1,14 +1,9 @@
-const FilterReducer = (state = [], action) => {
-  let changedState = [...state];
-
-  console.log(action.payload);//eslint-disable-line
-  function FilterBook(book) {
-    return book.category === action.payload;
-  }
+const FilterReducer = (state = 'All', action) => {
+  let changedState = state;
 
   switch (action.type) {
     case 'Change':
```

Comment:
- [ ] Please change this to `CHANGE_FILTER`.  In fact, it is always better to follow the best pratices